src/java.xml/share/classes/com/sun/org/apache/xalan/internal/xsltc/compiler/XSLTC.java

```diff
@@ -18,6 +18,12 @@
  * limitations under the License.
  */
 
+/*
```

Comment:
We can omit the IBM copyright since it's a direct backport. Pls backport the `@LastModified:` change as well.

Reply:
Updated as suggested above.